pyqtgraph/colormap.py

```diff
@@ -104,7 +104,6 @@ def _getFromFile(name):
         else:
             csv_mode = False
         for line in fh:
-            name = None
```

Comment:
This is accidentally left over from the original idea of naming individual colors. Removing this line ensures that the returned color map is named correctly and not None).